SasquatchMac/SasquatchMacObjC/AppDelegate.m

```diff
@@ -19,6 +19,12 @@ @implementation AppDelegate
 - (void)applicationDidFinishLaunching:(NSNotification *)notification {
   [MSAppCenter setLogLevel:MSLogLevelVerbose];
 
+  // Set user id.
+  NSString *userId = [[NSUserDefaults standardUserDefaults] objectForKey:@"userId"];
```

Comment:
Define a constant for "userId". You can see an example in iOS test app [here](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/Sasquatch/Sasquatch/Constants.h#L32) and [here](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/Sasquatch/SasquatchObjC/AppDelegate.m#L117)

Reply:
I can see hard-coded "userId" in several places. Please address those, too.